CHANGES.rst

```diff
@@ -415,6 +420,8 @@ gaia
   epoch photometry service to return all data associated to a given source.
   [#2376]
 
+- New retrieval types for datalink (Gaia DR4 release). [#3110]
```

Comment:
This is at the wrong location, but I'm cleaning up the changelog for release time anyway, so it doesn't matter here.